WangscapeTest/TestMetaOutput.cpp

```diff
@@ -12,17 +12,18 @@
 #include <tilegen/TilesetGenerator.h>
 #include <tilegen/partition/TilePartitionerSquares.h>
 
-class TestMetaOutput : public ::testing::Test {
+#include "TestRequiringOptions.h"
+
+class TestMetaOutput : public TestRequiringOptions
+{
 protected:
-    std::string filename;
+    using TestRequiringOptions::TestRequiringOptions;
```

Comment:
Yes, that's probably better. I would prefer to keep the guard which prevents setOptionsFilename from being successfully called more than once, though; GTest tests are supposed to be unable to affect each other.